### ISSUE TITLE
OpenAlias integration

### ISSUE THREAD

**Issue by nickrod** [GrinPlusPlus/GrinPlusPlus#139]
Now that the slatepack address is standardized, would it be possible to integrate OpenAlias with the wallet? So I can send to something like 'user@pay.me' and it will automatically resolve to the correct grin address. https://openalias.org/index.html

**Comment by davidtavarez** [comment]
I like this, the only issue I see is that the wallet should be open to receive grin without user interaction... any "solution" for that?

**Comment by nickrod** [comment]
I don't think it would be necessary for receiving, this would only be for getting the address when sending and resolving that automatically instead of copy and paste or using an address book. You can see an example implementation with Bitcoin Electrum.

**Comment by DavidBurkett** [comment]
This is a cool idea @nickrod. I don't have time to integrate anything like this, but I'll leave this issue open in case anyone out there gets ambitious and feels like contributing.
